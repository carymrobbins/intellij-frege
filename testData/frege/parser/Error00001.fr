module Error00001 wher

cats = "dogs"
