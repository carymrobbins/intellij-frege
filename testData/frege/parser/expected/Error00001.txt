Frege file
  MODULE_DECL
    PsiElement(MODULE)('module')
    MODULE_REF
      MODULE_NAME
        PsiElement(IDENT)('Error00001')
    PsiErrorElement:Expected WHERE
      <empty list>
    PsiElement(IDENT)('wher')
  PsiElement(EOL)('\n')
  PsiElement(IDENT)('cats')
  PsiElement(=)('=')
  PsiElement(STRING_LITERAL)('"dogs"')